dev-workflow-ui/webContent/includes/template/notifications.xhtml

```diff
@@ -37,11 +37,18 @@
         <p:dataScroller id="notifications" lazy="true" styleClass="notification-scroll" value="#{notificationBean.dataModel}" var="notification" chunkSize="20" mode="inline">
           <div class="#{notification.style} notification-container">
             <div class="p-d-flex p-jc-between">
-              <h:outputText id="notificationMessage" value="#{notification.message}" escape="false"/>
+              <h:outputLink id="notificationMessage" value="#{notification.infoAction.link()}" styleClass="notification-message">
+                <h:outputText value="#{notification.message}"/>
```

Comment:
still use escape=false if the template wants to make something <b> or color=red. etc.